starknet_forge_template/Scarb.toml

```diff
@@ -0,0 +1,16 @@
+[package]
+name = "{{ PROJECT_NAME }}"
+version = "0.1.0"
+
+# See more keys and their definitions at https://docs.swmansion.com/scarb/docs/reference/manifest
+
+[[target.starknet-contract]]
+sierra = true
+casm = true
+
+[dependencies]
+starknet = "2.2.0"
+snforge_std = { git = "https://github.com/foundry-rs/starknet-foundry.git", tag = "v0.5.0" }
```

Comment:
Git tag fallacy xD It is of course desired to have this tag for users, no doubt. However it will make testing the template impossible if something changes in basic declare-deploy flow. We should probably reuse the solution we have for git dependency (@cptartur is improving it rn) and replace the tag with branch in our test instead.

Also it would be great to fill this tag with current version of forge from the main `Cargo.toml`, although not sure if it's possible together with `include_dir` (it would have to be modified before `include_dir` executes)

Reply:
I changed how it works, so currently I use `scarb add -git .../starknet-foundry.git` to get the the lastest released version